ci: Test

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13007942271)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/389
<!-- Follow Merge -->
Hi @nikitabelonogov!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/389 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12904954066)

nikitabelonogov committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12904954066

nikitabelonogov commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13007942271)

nikitabelonogov committed:
Merge branch 'develop' into 'fb-ci-test'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13007942271

nikitabelonogov committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13007942271
